src/aimcore/web/ui/package.json

```diff
@@ -79,8 +79,8 @@
     "zustand": "^4.1.1"
   },
   "scripts": {
-    "start": "cross-env SKIP_PREFLIGHT_CHECK=true node --no-deprecation node_modules/react-scripts/scripts/start.js craco --max_old_space_size=6144 start",
-    "build": "cross-env SKIP_PREFLIGHT_CHECK=true node --no-deprecation node_modules/react-scripts/scripts/build.js craco --max_old_space_size=6144 build && gzipper c -i js,css,html ./build && node tasks/index-html-template-generator.js",
+    "start": "cross-env SKIP_PREFLIGHT_CHECK=true node --no-deprecation node_modules/react-scripts/scripts/start.js craco --max_old_space_size=4096 start",
+    "build": "cross-env SKIP_PREFLIGHT_CHECK=true node --max_old_space_size=4096 --no-deprecation node_modules/react-scripts/scripts/build.js craco --max_old_space_size=4096 build && gzipper c -i js,css,html ./build && node tasks/index-html-template-generator.js",
```

Comment:
let's remove the duplication of --max_old_space_size=4096 from build script

Reply:
@KaroMourad both commands need the option, that's why the build was failing previously